cmd/gpu-kubelet-plugin/deviceinfo.go

```diff
@@ -199,5 +200,8 @@ func (d *MigDeviceInfo) GetDevice() resourceapi.Device {
 			Value: *resource.NewQuantity(1, resource.BinarySI),
 		}
 	}
+	if d.pcieRootAttr.Name != "" {
```

Comment:
Can we make `d.pcieRootAttr` a pointer so we can check `nil` here?

Reply:
Yep, made that change.